### PR TITLE
Add ARK: Survival Evolved (generic)

### PR DESCRIPTION
**Pull request by Greelan**
Adds ARK: Survival Evolved

Associated comments: https://discord.com/channels/266012086423912458/1046205513970495538/1048454006831325244

### CODE REVIEW

**Comment by IceOfWraith** [comment]
You're the best. <3

**Review by IceOfWraith — CHANGES_REQUESTED** [review]
This was a beast to review. But I gotta say, you did an EXTREMELY good job on it. The mod handling is a thing of dreams. I don't know of any provider that offers this level of ease for Ark. Very nice work.

I didn't notice any typos or formatting issues. There are two items I've put for feedback/changes though.

File: ark-seconfig.json [hunk 1]
```diff
@@ -0,0 +1,2165 @@
...
+        "FieldName":"SpectatorPassword",
+        "InputType":"password",
+        "IsFlagArgument":false,
+        "ParamFieldName":"SpectatorPassword",
+        "IncludeInCommandLine":true,
+        "DefaultValue":"",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Player Limit",
```
[the remark this answers is not in the document]
Comment (Greelan) [reply ark-seconfig.json:234]: Done

File: ark-seconfig.json [hunk 2]
```diff
@@ -0,0 +1,2165 @@
...
+        "IsFlagArgument":false,
+        "ParamFieldName":"MaxPlayers",
+        "IncludeInCommandLine":true,
+        "DefaultValue":"70",
+        "Placeholder":"70",
+        "Suffix":"players",
+        "EnumValues":{}  
+    },
+    {
+        "DisplayName":"MultiHome IP Binding",
```
[the remark this answers is not in the document]
Comment (Greelan) [reply ark-seconfig.json:250]: That's the default when it is left blank.

**Comment by Greelan** [comment]
Thanks for doing the review - 'twas a beast to review, and more of a beast to write. I'd say the most involved template I've done. Hopefully the community sees value in it.

**Review by IceOfWraith — DISMISSED** [review]
They definitely will.

**Comment by Greelan** [comment]
Will wait for 2.4.1.4 to drop before merging, as it needs some elements currently in Nightly and the MinAMPVersion does not actually prevent an instance being created on an earlier version.

**Comment by IceOfWraith** [comment]
Mike will also want to mark the other Legacy when he releases.

**Review by IceOfWraith — APPROVED** [review]
Yes, yes I did. *smug face*

**Pull request merged**